.github/workflows/refapp-3x-clinical-visit.yml

```diff
@@ -10,6 +10,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./qaframework-bdd-tests
```

Comment:
```suggestion
  run:
      working-directory: ./qaframework-bdd-tests
```

Reply:
Sorry, I didn't catch that suggestion. Could you please explain?

I referred to the following example on https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun when setting the `working-directory`.
![image](https://user-images.githubusercontent.com/27498587/148049040-4bbf0663-f737-4c92-8198-753aaf4331fc.png)

Reply:
@Piumal1999 your point is right and thanks for sharing the document ion on the subject.